resources/tests/csvs/voteplans.csv

```diff
@@ -0,0 +1,3 @@
+id,chain_voteplan_id,chain_vote_start_time,chain_vote_end_time,chain_committee_end,chain_voteplan_payload,fund_id
```

Comment:
Depending on when https://github.com/input-output-hk/vit-servicing-station/pull/79 is merged
`chain_committee_end` -> `chain_committee_end_time`

Reply:
Lets merge #79 first then and I'll rebase here.

Reply:
Done